From 881a901fc9ff65e1b4e837e8645a4a9e0dc6948b Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Tue, 15 Oct 2019 12:29:05 +0200
Subject: [PATCH 0010/1239] arm64: dts: armada-8040-mcbin: set cp1 comphy to
 SGMII2.5 mode

Currently, cp1 comphy5 is set to SGMII3 - this is incorrect.
According to mcbin board layout, Comphy5 should be set to
SGMII2 2.5G.

Change-Id: Ia7e5c670354645681b04387252cda5d9a09affde
Signed-off-by: Igal Liberman <igall@marvell.com>
---
 arch/arm/dts/armada-8040-mcbin.dts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/armada-8040-mcbin.dts b/arch/arm/dts/armada-8040-mcbin.dts
index 5a046d9de4..eb75387ce9 100644
--- a/arch/arm/dts/armada-8040-mcbin.dts
+++ b/arch/arm/dts/armada-8040-mcbin.dts
@@ -283,7 +283,7 @@
 	 * Lane 2: USB HOST 0
 	 * Lane 3: SATA1
 	 * Lane 4: SFI (10G)
-	 * Lane 5: SGMII3
+	 * Lane 5: SGMII2
 	 */
 	phy0 {
 		phy-type = <PHY_TYPE_SGMII1>;
@@ -302,6 +302,7 @@
 		phy-type = <PHY_TYPE_SFI>;
 	};
 	phy5 {
-		phy-type = <PHY_TYPE_SGMII3>;
+		phy-type = <PHY_TYPE_SGMII2>;
+		phy-speed = <PHY_SPEED_3_125G>;
 	};
 };
-- 
2.29.0

